app/models/author.rb

```diff
@@ -14,4 +14,14 @@ class Author < ApplicationRecord
   belongs_to :thesis
 
   validates :graduation_confirmed, exclusion: [nil]
+
+  # Given a row of CSV data from registrar import, update graduation
+  # confirmed attribute (if needed) to true or false based on CSV data
+  def set_graduated_from_csv(row)
+    graduated = row['Degree Status'] == 'AW'
```

Comment:
Yeah, I get the nervousness there and for good reason. But this is in fact the logic we want to use, because we also want to set `graduation_confirmed` status to `false` if it was set `true` in the database and the CSV data says otherwise (which would include any value other than 'AW').